roles/preflight/README.md

```diff
@@ -158,129 +242,220 @@ The idea is use one of them.
+
+        # Optional; provide this token when using create_pr option
+        # while creating operator or helm-chart project
+        # Check the instructions below on how to create it
+        github_token_path: "/opt/cache/dcicertbot-token.txt"
+
+        # Mandatory for disconnected environment,
+        # this registry is used for mirrored images
+        # and to store an index (catalog) image.
+        dci_local_registry: registry.local.lab:4443
+
+        # Optional, provide it if your registry is self-signed.
+        preflight_custom_ca: "/var/lib/dci-openshift-agent/registry/certs/cert.ca"
+
+        # List of operators to certify,
+        # you could provide many operators at once.
+        preflight_operators_to_certify:
+          - bundle_image: "quay.io/rh-nfv-int/bla-bla-bundle:v0.2.1"
+            # Mandatory for the connected environments.
+            index_image: "quay.io/rh-nfv-int/bla-bla-catalog:v0.2.1"
+            # Optional; provide it when you need to create
+            # a new "Container Image project" and submit test results in it.
+            create_container_project: true
+            # Optional; provide it when you need to create
+            # a new "Operator Bundle Image" and submit test results in it.
+            create_operator_project: true
+            # Required when creating cert project
+            short_description: "Add 50+ characters image description here"
+            # Optional; use it to automatically open cert PR
+            # at the certified-operators repository
+            create_pr: true
+            # Optional; use it to automatically merge cert PR
+            # at the certified-operators repository
+            merge_pr: true
+            # Product Listings to attach to the cert project (Optional)
+            pyxis_product_lists:
+              - "yyy"
+              - "xxx"
+        # Project certification setting (Optional)
+        # This allows to fill the rest of the project settings after project creation
+        # Any project for containers images certifications can use them
+        # TODO: provide cert_settings example for every project type
+        cert_settings:
+          # Operator and Container
+          auto_publish: false
+          registry_override_instruct: "These are instructions of how to override settings"
+          email_address: "email@email.com"
+          application_categories: "Networking"
+          privileged: false
+          repository_description: "This is a test repository"
+          build_categories: "Standalone image"
+          os_content_type: "Red Hat Universal Base Image (UBI)"
+          release_category: "Generally Available"
+            
+      use_previous_topic: true
+      inputs:
+        kubeconfig: kubeconfig_path
+    ...
     ```
 
 - Run dci-openshift-app-agent:
 
     ```bash
-    dci-openshift-app-agent-ctl -s -- -v
+    KUBECONFIG=$KUBECONFIG dci-pipeline-schedule testpmd-operator-e2e-disconnected
```

Comment:
This is just to run the second case (disconnected), but not valid for the first case